### DISCUSSION TITLE
Question: Trino’s release process and version policy

### DISCUSSION THREAD

**Discussion by dmyar21** [trinodb/trino#20032] [Q&A] [answered]
Hi colleagues,
What is Trino’s release process and version policy?
How can we determine which versions of Trino are backward compatible and which ones are not?

**Comment by mosabua** [comment, marked as answer]
Here are a few things:

* we try to release every Wednesday, release process itself is a large scope and mostly internal to the maintainers
* there is no semantic or whatever versioning, incremental versions only

In terms of compatibility .. every version potentially includes breaking changes. Since 432 we are flagging them in the release notes. If something actually breaks for your usage however is a very different question and depends a LOT on connectors and other usage and the version jump specifically. 

On the other hand we try to avoid breaking changes as much as possible and introduce deprecated configs and so on. Things like the JDBC driver and the client api are very stable and have a lot of compatibility across versions.

In the end you need to test yourself for your deployment scenarios and usage. We can not make any guarantees from one version to another.
